### PR TITLE
Support node selector for JobManager and TaskManager pods

### LINKED ISSUE #10: Support node selector for JobManager and TaskManager pods

### PR DESCRIPTION
**Pull request by functicons**
Fix #10

### CODE REVIEW

File: config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml [hunk 1]
```diff
@@ -1830,6 +1830,12 @@ spec:
                     - mountPath
                     type: object
                   type: array
+                nodeSelector:
+                  additionalProperties:
+                    type: string
+                  description: 'Selector which must match a node''s labels for the
```
Comment (hongyegong) [inline config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml]: node's plz.

File: config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml [hunk 2]
```diff
@@ -3118,6 +3124,12 @@ spec:
...
```
Comment (hongyegong) [inline config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml]: same as above.

File: config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml (hunk 1 above)
Comment (functicons) [reply config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml]: This is generated, the whole string is inside of a pair of single quote (').

File: config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml (hunk 2 above)
Comment (functicons) [reply config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml]: see above.

**Pull request merged**